.github/workflows/smoke-test.yml

```diff
@@ -66,16 +79,16 @@ jobs:
       - name: Start minikube (1st boot)
         run: |
           set -ex
-          ./out/minikube start --driver=${{ matrix.driver }} --memory 8gb --no-kubernetes --alsologtostderr
+          ./out/minikube start --driver=${{ matrix.driver }} ${{ matrix.network_flag }} --memory 8gb --no-kubernetes --alsologtostderr
       - name: Stop minikube
         run: |
           set -ex
           ./out/minikube stop
       - name: Start minikube again (2nd boot)
         run: |
           set -ex
-          ./out/minikube start --driver=${{ matrix.driver }} --no-kubernetes --alsologtostderr
+          ./out/minikube start --driver=${{ matrix.driver }} ${{ matrix.network_flag }} --no-kubernetes --alsologtostderr
```

Comment:
On the second start we don't need any argument. We know that these arguments are ignored (not sure about --no-kubernetes).

Reply:
yeah I had it without args at first I wanted to double check that is not casuing the failure